### DISCUSSION TITLE
Need help. Server 500 after enabling eCommerce Framework Bundle

### DISCUSSION THREAD

**Discussion by DrGreenCodeWizard** [pimcore/pimcore#13670] [Q&A / Support]
Hi we just set up a new pimcore installation and after I have clicked on the plus button to enable the EcommerceFrameworkBundle I got a error message which said something like "cannot save" then the page reloaded and brought me a 500...now I´m stuck.

Can anyone help me out please?

![WhatsApp Image 2022-11-22 at 22 35 13](https://user-images.githubusercontent.com/58132610/203697129-46012b5f-dacd-416c-80a1-be96a694a508.jpeg)

**Comment by kingjia90** [comment]
It is hard to tell what is causing the problem, is the `APP_ENV=dev` under root `.env` file? This activates the developer env and would display more details.
Also try to run `bin/console cache:clear` in the terminal, likely it would appear an error message
